### DISCUSSION TITLE
<i would like to host a podcast on this platform>

### DISCUSSION THREAD

**Discussion by Rrrobbb** [community/community#46240] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

hello!
i'm new here, having looked forbthis page after reading a New Yorker article speaking about how transcription services can be done in new-fangled ways.
No that i'm here Nd can see how cool this place is, i was wondering if i can upload a podcast for people to listen to?
i already have the first episode recorded (and 'published' on Acast, i think it's about 50-something megabytes?
thanks for any advice you can provide :-)
thanks,
Rob

**Comment by Akash1134** [comment]
Hi @Rrrobbb  , we’re glad you’re here 😀

Thanks for sharing your project  with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
